### ISSUE TITLE
Document database export and import

### ISSUE THREAD

**Issue by kaligrafy** [chairemobilite/transition#6]
create a zip with sql dump and cache data (at least auto run create cache after import)

**Comment by greenscientist** [comment]
@kaligrafy isn't this more like a infra issue ? People should know how to backup their database I think. 
Maybe we could add something in the documentation ?

**Comment by kaligrafy** [comment]
It is mostly for developpers and people installing it locally. I tnd to always forget the correct settings when exporting (like the ownership and schemas correct config)

**Comment by kaligrafy** [comment]
The goal is to be able to do a full backup of a Transition instance with users and prefs, and then re-import it on nother instance, which make sure everything is identical on the new machin/instance

**Comment by kaligrafy** [comment]
yes, it could be different then a simple dump/load so we can make sure it is always working on any system
